woocommerce/woocommerce-gutenberg-products-block
# Bump @wordpress/env to 8.2.0

## What

This PR is a follow up of https://github.com/woocommerce/woocommerce-blocks/pull/9875. @dinhtungdu noticed that after upgrading `@wordpress/env` to 8.1.1, looks like the test environment for Playwright tests is becoming more flaky. For this reason, we want to see `@wordpress/env@8.2.0` can improve the flakiness.  
## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that the E2E PW tests are green.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
